Python/importlib_zipimport.h

```diff
@@ -1,121 +1,122 @@
 /* Auto-generated by Programs/_freeze_importlib.c */
```

Comment:
Hum, we should mark these generated files as binary in .gitattributes to try to hide them in reviews. I dont ask you to do that in your PR.